test/test_gmpy2_mpz_misc.txt

```diff
@@ -652,9 +652,32 @@ TypeError:
 mpz(3)
 >>> gmpy2.next_prime(2)
 mpz(3)
+>>> gmpy2.next_prime(1000000)
+mpz(1000003)
 >>> gmpy2.next_prime(2357*7069-1) == 2357*7069
 False
 
+Test prev_prime
+---------------
+>>> gmpy2.prev_prime('a')
+Traceback (most recent call last):
+  ...
+TypeError:
+>>> gmpy2.prev_prime(2)
+mpz(2)
+>>> gmpy2.prev_prime(mpz(3))
+mpz(2)
+>>> gmpy2.prev_prime(3)
+mpz(2)
+>>> gmpy2.prev_prime(1000004)
+mpz(1000003)
+>>> gmpy2.prev_prime(1000033)
+mpz(1000003)
+>>> gmpy2.prev_prime(-100)
+mpz(2)
+>>> gmpy2.prev_prime(1)
+mpz(2)
+
```

Comment:
Per #377, you shouldn't use test/*.txt files for new tests.

These tests could be moved to the test_mpz.py, I think.  And you could use the pytest's skipif fixture to skip them if the GMP library is too old.

Reply:
Done

Reply:
Some *.txt are there.

Reply:
I left one additional test for next_prime(1000000) which I think is a nice test I can remove if you don't want it added here.